modules/installation-extend-edge-nodes-aws-local-zones.adoc

```diff
@@ -9,11 +9,11 @@ endif::[]
 :_content-type: PROCEDURE
 [id="installation-extend-edge-nodes-aws-local-zones_{context}"]
 = Creating user workloads in AWS Local Zones
-After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create user workloads in Local Zone subnets. 
+After you create an Amazon Web Service (AWS) Local Zone environment, and you deploy your cluster, you can use edge worker nodes to create user workloads in Local Zone subnets.
 
-After the `openshift-installer` creates the cluster, the installation program automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod, or deployment, to a node if the pod does not match the specified tolerations for a taint. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
+After the installation program creates the cluster, the program automatically specifies a taint effect of `NoSchedule` to each edge worker node. This means that a scheduler does not add a new pod, or deployment, to a node if the pod does not match the specified tolerations for a taint. You can modify the taint for better control over how each node creates a workload in each Local Zone subnet.
```

Comment:
```suggestion
After you run the installation program and create the cluster, a taint effect of `NoSchedule` is automatically specified to each edge worker node. This means that a scheduler does not add a new pod, or deployment, to a node if the pod does not match the specified tolerations for a taint. You can modify the taint for better control over how nodes create workloads in each Local Zone subnet.
```

Reply:
I tried my hand at rewriting this a bit to avoid some anthropomorphism: https://www.ibm.com/docs/en/ibm-style?topic=grammar-anthropomorphism

Reply:
Anthropomorphism would not apply here because the installation program does the tasks: 

![Screenshot from 2023-09-12 15-21-52](https://github.com/openshift/openshift-docs/assets/57954076/b2a84ba7-e9a4-43c8-ae7d-06ca84b62d61)